Resources/Prototypes/Entities/Objects/Weapons/Throwable/bola.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  name: bola
+  parent: BaseItem
+  id: Bola
+  description: Tied together with some spare cable and metal.
+  components:
+  - type: Item
+    size: 3
```

Comment:
3 is too low for these

Reply:
For keron: if it's parent is baseitem it will inherit size 5 so you can remove the whole item component here since it's only defining size, to prevent duplicating it.

Reply:
well it should be more than 5 too

Reply:
Fixed.